src/main/java/com/czertainly/core/service/cmp/mock/KeystoreService.java

```diff
@@ -0,0 +1,147 @@
+package com.czertainly.core.service.cmp.mock;
+
+import org.springframework.util.ResourceUtils;
+
+import javax.crypto.SecretKey;
+import java.io.*;
+import java.security.*;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import java.util.*;
+
+public class KeystoreService {

```

Comment:
What is `KeystoreService`? It seems to me like the helper class for unit testing and it should not be here.

Reply:
DONE, using in mock (moved together into package test)